src/link/edit.js

```diff
@@ -193,18 +210,33 @@ const Edit = ( {
 		}
 	}, [ value ] );
 
+	useEffect( () => {
+		const tooltipDisplayedCount = parseInt( wikipediapreviewCustomTooltip.tooltipCount );
+
+		if ( tooltipDisplayedCount < tooltipDisplayedLimit ) {
+			waitOneSecThenDisplayTooltip();
+		}
+	} );
+
```

Comment:
do you think it's possible to move this `useEffect`, `waitOneSecThenDisplayTooltip` method, and also all tooltip related state into `src/link/tooltip.js`? and just leave <CustomTooltip /> here

for it's easier to know all of these are for the custom tooltip scope.

Reply:
That's an excellent idea, thanks. Check out https://github.com/wikimedia/wikipediapreview-wordpress/pull/111/commits/51680afc81cc3460a81091d557e2026a56f50b89